devops/openshift/api-deploy.yml

```diff
@@ -240,6 +237,17 @@ objects:
       minReplicas:  "${{REPLICAS}}"
       maxReplicas: 10
       targetCPUUtilizationPercentage: 80
+  - apiVersion: policy/v1
+    kind: PodDisruptionBudget
+    metadata:
+      name: ${NAME}-pdb
+    spec:
+      selector:
+        matchLabels:
+          app: ${NAME}
+      minAvailable: 2
```

Comment:
As per openshift/kubernetes, `maxUnavailable` and `minAvailable` are mutually exclusive. Should we use both?

![image](https://github.com/bcgov/SIMS/assets/54600590/d9dd6cd5-e42e-403b-aebd-4f9bb2a26f32)

Reply:
Yes, we can use both minAvailable and maxUnavailable in a PodDisruptionBudget, in our current scenario we set minAvailable to 2 and maxUnavailable to 1, it ensures there are always at least 2 pods available, and only 1 pod can be undergoing disruption at any given time.

Reply:
Not sure if my understanding is right, but document says only one of the properties can be used for a single pdb.